### PR TITLE
Add an ADMINISTER command to do reindex

### PR DESCRIPTION
**Pull request by msullivan**
I had originally called it `reindex_object`, but remembered that
Elvis wanted `NOUN_VERB` names for these functions, so I made it
`object_reindex`.

Verified that it does work to get pgvector indexes that were
created too early working.

### CODE REVIEW

File: edb/server/compiler/compiler.py [hunk 1]
```diff
@@ -1449,8 +1506,10 @@ def _compile_ql_administer(
             )
 
         return dbstate.MaintenanceQuery(sql=(b'ANALYZE',))
-    elif ql.expr.func == 'repair_schema':
+    elif ql.expr.func == 'schema_repair':
         return ddl.administer_repair_schema(ctx, ql)
+    elif ql.expr.func == 'object_reindex':
```
Comment (elprans) [inline edb/server/compiler/compiler.py:1511]: Why `object_reindex` and not just `reindex`?  Is this somehow specific to object relations or should also apply to link relations (e.g. indexes on link properties).

**Comment by msullivan** [comment]
I didn't call it "reindex", because "reindex" is reserved. Obviously that
makes no sense. I'll change it.

**Comment by 1st1** [comment]
Yeah, let's call it "reindex", other than that lgtm!

**Review by elprans — COMMENTED** [review]
What if I want to reindex a specific index?

**Comment by msullivan** [comment]
> What if I want to reindex a specific index?

That's not supported currently.

I'm actually not sure if there is any particularly way to specify a specific index while using function call syntax. For DDL ALTERs/DROPs we recapitulate the definition, basically, and that wouldn't be syntactically valid here. If we were doing custom syntax, it would be easy, of course.

One option would be to support specifying pointers, and then reindex all indexes that reference that pointer.

**Review by 1st1 — APPROVED** [review]

**Review by elprans — APPROVED** [review]

**Review by elprans — APPROVED** [review]

**Pull request merged**